src/features/form/components/Legend.tsx

```diff
@@ -47,12 +47,7 @@ export function Legend(props: IFormLegendProps) {
   return (
     <>
       <div className={classes.legendHelpTextContainer}>
-        <label
-          className='a-form-label title-label'
-          htmlFor={props.id}
-        >
-          {LabelText}
-        </label>
+        <legend className='a-form-label title-label'>{LabelText}</legend>
```

Comment:
Would be nice to get rid of global css whenever we touch code where it is present

Reply:
Agree, i´ll get on it 👍